components/schema-migrator/migrations/director/20231211174610_add_aspect_event_resource.up.sql

```diff
@@ -0,0 +1,125 @@
+BEGIN;
+
+-- Drop views --
+DROP VIEW IF EXISTS aspect_event_resources_subset;
+DROP VIEW IF EXISTS aspect_event_resources;
+DROP VIEW IF EXISTS aspects_tenants; -- this view will not be changed but it depends on the column aspects.event_resources; that's why the view has to be dropped and re-created in order to drop the column
+DROP VIEW IF EXISTS tenants_aspects;
+
+-- Create aspect_event_resources table
+CREATE TABLE aspect_event_resources (
```

Comment:
Should we consider implementing an e2e test in the ord aggregator which gets integration dependency with expanded aspects and aspect event resources?